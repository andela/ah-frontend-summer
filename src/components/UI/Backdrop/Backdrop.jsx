import React from "react";
import "../../../assets/styles/Backdrop.scss";

const backdrop = (props) => (
    props.show ? <div className="Backdrop" onClick={props.clicked}></div> : null
);

export default backdrop;
