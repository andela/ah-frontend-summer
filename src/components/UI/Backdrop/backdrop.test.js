import React from 'react';
import { shallow } from 'enzyme';

import Backdrop from './Backdrop';

describe('<Backdrop />', () => {
    it('Backdrop will render without crashing', () => {
        let wrapper = shallow(<Backdrop />);
        expect(wrapper).toMatchSnapshot();
    });
});
