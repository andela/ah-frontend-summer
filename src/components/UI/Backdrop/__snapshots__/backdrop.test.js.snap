// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Backdrop /> Backdrop will render without crashing 1`] = `""`;
